packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -132,13 +132,25 @@ export class IgnitionModuleBuilderImplementation<
       ResultsContractNameT,
       IgnitionModuleResultsT
     >,
-    public readonly chainId: number,
-    public readonly accounts: string[],
     public readonly parameters: ModuleParameters = {}
   ) {
     this._futureIds = new Set<string>();
   }
 
+  public getAccount(accountIndex: number): AccountRuntimeValue {
+    return new AccountRuntimeValueImplementation(accountIndex);
+  }
+
+  public getParameter<ParamTypeT extends ModuleParameterType>(
+    parameterName: string,
+    defaultValue?: ParamTypeT
+  ): ModuleParameterRuntimeValue<ParamTypeT> {
+    return new ModuleParameterRuntimeValueImplementation(
+      parameterName,
+      defaultValue
+    );
+  }
+
```

Comment:
We return `RuntimeValue`s now.

Reply:
`getAccount` instead of `accounts`, as we can't pre-populate an array of `AccountRuntimeValue`.